cmd/cmd.go

```diff
@@ -114,10 +114,6 @@ func RunHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	mp := server.ParseModelPath(args[0])
-	if err != nil {
-		return err
-	}
-
```

Comment:
Unrelated but it's been bothering me

Reply:
good cleanup

Reply:
is this just leftover cruft? looks good to get rid of it.